src/Repeater/ServerRepeaterLauncher.ts

```diff
@@ -123,6 +124,7 @@ export class ServerRepeaterLauncher implements RepeaterLauncher {
       );
     });
     this.repeaterServer.errorOccurred(({ message }) => {
+      captureException(message);
```

Comment:
Why don't put it under the `logger.error` method to avoid duplicates?

Reply:
Ok, let's capture exceptions only there, since other solutions will require code duplication or big refactoring.

Initially I thought about 2 potential problems:

1. It will affect other parts of bright cli, not only repeater,
2. No stack trace information.

Regarding the first problem, I already started thinking that it's a good side effect, so we can live with that.

But the second one is tricky. If we use `logger.error` to capture "exceptions", we will loose error stack trace, since this function accepts only strings:

1. Absence of stack trace will lead to improper issues grouping inside Sentry. But, in our case, I think it can be solved by adding fingerprints,
2. We will have less information about where error occurred - we will capture only places where it was logged.

Unfortunately commands catch errors and don't rethrow them in order to have a single point of handling errors.

So, as I see we have 3 options for now:

1. Refactor commands in order to have a single handler to catch exceptions + manually add catchExceptions to code like `.on('error, () => {})`,
2. Capture exceptions in `logger.error` only. This will easily allow us to cover all cases, but with some information loose.
3. Add capture logic manually to crucial places.

So let's go with the #2 for now.

Reply:
The common approach involves utilizing the `captureMessage` function for error messages and manually invoking `captureExecution` for silently handled errors in order to retain the stack trace and fingerprint resolution. However, it depends on the context.

We can also adjust the `error` method signature by adding the overloading to highlight the difference:

```ts
public error(message: string, error: Error): void;
public error(message: string, ...args: unknown[]): void;
```

Reply:
The example with error method signature is good, thanks. I did similar thing, but placed error in the beginning to  preserve string formatting behaviour